be/app/Http/Controllers/AuthController.php

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
- delete these files in your root folder. You mistakenly installed a laravel package in the root folder instead of inside the `BE`
![image](https://user-images.githubusercontent.com/73863685/219265506-54a0254e-cb67-44c5-9e56-29113b68224b.png)